lib/PHPExif/Mapper/Exiftool.php

```diff
@@ -214,7 +237,7 @@ public function mapRawData(array $data)
                     }
 
                     break;
-                case self::DATETIMEORIGINAL_QUICKTIME:
+                case self::DATETIMEORIGINAL_APPLE:
```

Comment:
I think, there is a testcase for "DATETIMEORIGINAL_QUICKTIME" which needs to be adjusted

Reply:
Good catch; I just pushed a commit to take care of it!